lib/pipeline.js

```diff
@@ -82,13 +82,17 @@ class PipelineModel extends BaseModel {
     formatScmUrl(scmUrl) {
         let result = scmUrl;
         const matched = (schema.config.regex.SCM_URL).exec(result);
+        let branchName = matched[4];
```

Comment:
Okay, I'll fix it.
Should I define that constant at `screwdriver-data-schema` module?